src/services/highcharts-factory/highcharts-factory.svc.coffee

```diff
@@ -38,7 +42,7 @@ angular
       angular.extend(@options, options)
       chartConfig = angular.merge({}, @template(), @formatters(), @todayMarker())
       if _.isEmpty(@hc)
-        @hc = Highcharts.chart(@id, chartConfig)
+        @hc = Highcharts.stockChart(@id, chartConfig)
```

Comment:
Will all Impac! angular charts use StockChart now?

Reply:
If I'm not wrong, stockChart is just like highCharts with more options, so I think this should be ok?..